recipes/lima-vm/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
+mkdir -p ${PREFIX}/bin
+mkdir -p ${PREFIX}/share
+
+# Add missing --force to codesign call on macOS
+sed -i "s/codesign --entitlements/codesign --force --entitlements/" Makefile
+
+make VERSION=${PKG_VERSION}
+cp -r _output/bin/* ${PREFIX}/bin
+cp -r _output/share/* ${PREFIX}/share
+
+mkdir -p ${PREFIX}/etc/bash_completion.d
+mkdir -p ${PREFIX}/share/zsh/site-functions
+mkdir -p ${PREFIX}/share/fish/vendor_completions.d
+limactl completion bash > ${PREFIX}/etc/bash_completion.d/lima
```

Comment:
Does this actually work? 🤯

Reply:
It definitely generates the completion files - I just tested that now locally.  Whether the users's shell properly adds them will probably depend on their specific configuration.  Did you run into issues with this in the previous PR?  Unfortunately the logs were expired so I wasn't able to see where it was failing.  If there are any tweaks to be made, please feel free to make an issue or PR on the feedstock!